src/client/app/common/views/components/reactions-viewer.vue

```diff
@@ -12,12 +12,20 @@ export default Vue.extend({
 	components: {
 		XReaction
 	},
+	data() {
+		return {
+			initialReactions: null as unknown as Set<string>
```

Comment:
`new Set(Object.keys(this.reactions))`じゃダメかな

Reply:
data関数が呼ばれた時点だと `this.reactions` は `undefined` になっています。
そのため `Object.keys(undefined)` で `TypeError: Cannot convert undefined or null to object` が発生します。

Reply:
うーむなんでだろう

Reply:
`this.note.reactions` にすれば解決する可能性はある(未検証)

Reply:
あー、this.reactionってpropではなくてcomputedでしたね
失礼しました

Reply:
これっぽい

https://forum.vuejs.org/t/computed-properties-in-data/11231/7

> Technically, this cannot work because during component creation, the data is evaluated before the computed array, so this.test doesn’t exist yet.